spec/src/main/asciidoc/01-introduction.adoc

```diff
@@ -0,0 +1,54 @@
+== Introduction
+
+=== Scope
+
+Jakarta Config ("Specification") is a Java API for working with configurations. It supports externalized configuration allowing applications to use different configurations for different environments (dev, test, prod), and allows reading data from different layered configuration sources such as property files, environment variables, etc.
```

Comment:
I understand this is just a sketch. As it exists, it needs a _lot_ of work. I will try to draft an introduction befitting a Jakarta EE specification. I will look at our discussion in https://github.com/eclipse-ee4j/config/discussions/142 to proceed.